saleor/graphql/account/mutations/account.py

```diff
@@ -322,7 +318,11 @@ def perform_mutation(cls, _root, info, **data):
 
 class AccountUpdateMeta(UpdateMetaBaseMutation):
     class Meta:
-        description = "Updates metadata of the logged-in user."
+        description = (
+            "DEPRECATED: Will be removed in Saleor 2.11."
```

Comment:
Graphene supports `deprecation_reason` for mutations at the level of attaching mutations as `Field` objects to the schema (in our case I think it's always in `schema.py` files). We don't have to change the description but instead in schema.py do:
```
account_update_meta = AccountUpdateMeta.Field(
    deprecation_reason="Will be removed in Saleor 2.11. Use the `updateMeta` mutation instead."
)
```

We need to do that for all mutations that we deprecate in this PR.